store/store.go

```diff
@@ -14,7 +14,6 @@ import (
 
 type StoreResult struct {
 	Data interface{}
-	Err  *model.AppError
 
 	// NErr a temporary field used by the new code for the AppError migration. This will later become Err when the entire store is migrated.
 	NErr error
```

Comment:
Two things:

* The comment is unnecessary now so we can get rid of it.
* Since `NErr`  is the only error field in the `StoreResult`, we could refactor this and rename it to Err (I can help with this if you prefer)

Reply:
Yes, I agree. we should do the opposite. Change `Err` to be `error`.

Reply:
Actually, this is the first one, that only remove the Err to verify it there are still usage of this, another one can do the renaming.

Reply:
Sounds good to me.

Reply:
@rvillablanca I think we can remove the comment in the other PR, is going to be more obvious what is going on.

Reply:
Sure